umap/static/umap/js/modules/rendering/template.js

```diff
@@ -282,3 +289,66 @@ class Wikipedia extends PopupTemplate {
     return body
   }
 }
+
+class Route extends TitleMixin(PopupTemplate) {
+  async renderBody(feature) {
+    if (feature.type !== 'LineString' || feature.isMulti()) {
+      return super.renderBody(feature)
+    }
+    let prev
+    let dist = 0
+    const data = []
+    const latlngs = feature.ui.getLatLngs()
+    const map = feature._umap._leafletMap
+    const properties = feature.extendedProperties()
+    for (const latlng of latlngs) {
+      if (!latlng.alt) {
+        continue
+      }
+      if (prev) {
+        dist = map.distance(latlng, prev)
+      }
+      data.push([latlng.alt, dist])
+      prev = latlng
+    }
+    const [root, { altitude, chart }] = Utils.loadTemplateWithRefs(`
+      <div>
+        <p>
+          ${translate('Distance:')} ${properties.measure} •
```

Comment:
Maybe we should put ":" outside of the translated string, so to better factorise we other translated strings ?